src/AppBundle/Controller/Report/ReportSubmissionController.php

```diff
@@ -149,14 +149,13 @@ public function getCasrecData(Request $request)
         $repo = $this->getRepository(EntityDir\Report\ReportSubmission::class); /* @var $repo EntityDir\Repository\ReportSubmissionRepository */
 
         $ret = $repo->findAllReportSubmissions(
-            $request->get('status'),
-            $request->get('q'),
-            $request->get('created_by_role'),
```

Comment:
does anything else in client or api call findAllReportSubmissions with the params being removed?